application.py

```diff
@@ -1,26 +1,100 @@
 import os
+import sys
+import time
 
-from flask import Flask, session
-from flask_session import Session
-from sqlalchemy import create_engine
-from sqlalchemy.orm import scoped_session, sessionmaker
+from users import *
+from flask import Flask, session, render_template, request,redirect,url_for
+from sqlalchemy import create_engine,desc
 
-app = Flask(__name__)
 
 
+app = Flask(__name__, static_url_path='/static')
+app.secret_key = 'khammam'
+
+
+# Check for environment variable
 if not os.getenv("DATABASE_URL"):
     raise RuntimeError("DATABASE_URL is not set")
 
+# Configure session to use filesystem
+app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+db.init_app(app)
+with app.app_context():
+    db.create_all()
```

Comment:
db.create_all() is used to create a table, I do not think that it should be run each time when the flask runs.